src/endpoints/getMatch.ts

```diff
@@ -88,6 +88,20 @@ export const getMatch = (config: HLTVConfig) => async ({
     winnerTeam = team2
   }
 
+  let result: string | undefined = undefined;
+  if (winnerTeam && team1 && team2) {
+    team1.scores = Number($('.team1-gradient')
+      .children()
+      .last()
+      .text());
+    team2.scores = Number($('.team2-gradient')
+      .children()
+      .last()
+      .text());
+
+    result = `${team1.scores} - ${team2.scores}`;
+  }
+
```

Comment:
What is the point of this change, the MapResult object already has results info?

Reply:
Yes, map results contain the result for every map, but not the match result, how many maps each team won. Eg. best of 3 match result should be 2-0, 2-1, 1-2 or 0-2 (if a draw is not possible). 

Sure you can parse it from MapResult object. But even MapResult object doesn't contain directly which team won the map, you have to parse it from result string. 

So this makes a lot easier to get this info.

Reply:
Maybe we can replace the string in the map result with an object that contains a parsed result: something like `{team1: 16, team2: 12}` instead of `16-12`. We'll need to handle overtimes though. I know I'm in no position to request that from you, but I think it's the right approach.

Thank you for contributing to this project btw :)

Reply:
That's why there is `team1.scores` and `team2.score` so the result string can be used as shorthand and now it follows the same logic as [MatchResult.ts](https://github.com/gigobyte/HLTV/blob/master/src/models/MatchResult.ts) model where the result is also a string.

Reply:
But `team1.scores` and `team2.scores` are only the final result of the match, for example `1-2` in a Best of 3.